scalr/provider.go

```diff
@@ -90,7 +93,13 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 		return nil, err
 	}
 
-	providerUaString := fmt.Sprintf("terraform-provider-scalr/%s", providerVersion.ProviderVersion)
+	var providerUaString string
+	if providerVersion.Branch == defaultBranch {
+		providerUaString = fmt.Sprintf("terraform-provider-scalr/%s", providerVersion.ProviderVersion)
```

Comment:
Can we set a variable to `providerVersion.ProviderVersion` or branch with replace and then have a single `fmt.Sprintf("terraform-provider-scalr/%s", X)` call?